src/dlgTriggerEditor.cpp

```diff
@@ -182,9 +187,26 @@ dlgTriggerEditor::dlgTriggerEditor( Host * pH )
     QSizePolicy sizePolicy5(QSizePolicy::Expanding, QSizePolicy::Expanding);
     mpSourceEditorArea->setSizePolicy( sizePolicy5 );
     pVB1->addWidget( mpSourceEditorArea );
-    QPlainTextEdit * editp = mpSourceEditorArea->editor;
-    editp->setWordWrapMode( QTextOption::NoWrap );
-    connect(editp,SIGNAL(cursorPositionChanged()), this, SLOT(slot_cursorPositionChanged()));
+    mpSourceEditor = mpSourceEditorArea->editor;
+    mpSourceEditor->setWordWrapMode( QTextOption::NoWrap );
+#if QT_VERSION >= 0x050200
```

Comment:
This is incorrect - it was introduced in 5.3 (https://doc.qt.io/qt-5/qplaintextedit.html#placeholderText-prop), so this did not compile for me using 5.2. Suggest updating CI to also build with minimal Qt version we support as CI did not catch this.